examples/set-annotation/resources.yaml

```diff
@@ -5,12 +5,7 @@ metadata:
 data:
   some-key: some-value
 ---
-apiVersion: dev.example.com/v1
-kind: MyResource
+apiVersion: v1
+kind: Namespace
```

Comment:
Why change it to use a Namespace?

Reply:
Because the simple input format doesn't have `fieldSpecs` so it's meaningless to add a CRD to the resources.